Tests/ExperimentIntegrationTests.xctestplan

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Why do you have two test plans? are both needed?

Reply:
Also related to test plans, I can't see them correctly on Xcode, even after cleaning and building... do you see them correctly on this (or another view?). It is not listed here, but I can still see it from Fennec -> Edit scheme, however, as you can see there, the test suite `ExperimentIntegrationTests` is not available for the test plan...

<img width="706" alt="Screenshot 2023-07-28 at 12 03 18" src="https://github.com/mozilla-mobile/firefox-ios/assets/1897507/00147738-4f3b-449c-8f00-5200f854df9b">

Reply:
Ill remove that older test plan

Reply:
Not sure whats going on, I don't know how to fix this 😞

Reply:
I explained that in a previous comment above: https://github.com/mozilla-mobile/firefox-ios/pull/15574#discussion_r1274746314

Reply:
oh .. let me try again cleaning my environment and cloning your branch

Reply:
I think the files are created correctly but we are missing a correct reference between them so that they appear on my side. Let's see if @clarmso can see them correctly though before looking at a solution